resource/css/rtl.css

```diff
@@ -0,0 +1,700 @@
+}
+
+/* Fix for a nasty chromium bug causing the fonts not to render before a redraw.
+ * https://code.google.com/p/chromium/issues/detail?id=336476
+ ************************************/
+
+/* mediaquery
+ ***********************************/
+@media (max-width: 800px) {
+  html[dir="rtl"] .content > .feedback-logo, .content > .about-logo {
+    right: 10px;
+  }
+  
+  html[dir="rtl"] .content > .feedback-box, .content > #about, .content > .credits, .content > .version {
+    margin-right: 120px;
+    margin-left: initial;
+  }
+
+  html[dir="rtl"] .welcome-box, .right-box {
+    float: right;
+  }
+
+  html[dir="rtl"] #vocabulary-list, #vocabulary-list-right {
+    float: left;
+    margin-right: 15px;
+    margin-left: initial;
+  }
+
+  html[dir="rtl"] .vocab-category > ul, .vocab-category > li {
+    right: 0;
+  }
+
+  html[dir="rtl"] .vocabulary-separator {
+    margin-right: 0;
+    margin-left: initial;
+  }
+
+}
+
+@media (max-width: 550px) {
+
+  html[dir="rtl"] #navigation {
+    right: 3.5%;
+    left: initial;
+  }
+  
+}
+
+@media (max-width: 640px) {
+
+  html[dir="rtl"] .header-float {
+    margin-right: 15px;
+  }
+  
+}
+
+@media (max-width: 800px) {
+
+  html[dir="rtl"] .right-box {
+    right: 0;
+    left:initial;
+  }
+
+}
+
+@media (max-width: 700px) {
+
+  html[dir="rtl"] #navi1, #navi2, #navi3, #navi4 {
+    padding-right: 0;
+    padding-left: initial;
+  }
+
+  html[dir="rtl"] #navigation {
+    right: 2.5%;
+    left: initial;
+  }
+
+  html[dir="rtl"] .frontpage #language {
+    padding-left: 0;
+    padding-right: initial;
+    left: 2.5%;
+    right:initial;
+    text-align: right;
+  }
+
+  html[dir="rtl"] #service-logo {
+    left: 3%;
+    right: initial;
+  }
+
+  html[dir="rtl"] .frontpage-spacing .header-float {
+    float: right;
+  }
+
+  html[dir="rtl"] #navigation {
+    padding-left: 0;
+    padding-right: initial;
+  }
+
+}
+
+@media (min-width: 640px) {
+  html[dir="rtl"] .voclist-wide > .right-box {
+    right: 0;
+    left: initial;
+  }
+
+}
+
+@media (min-width: 801px) {
+
+  html[dir="rtl"] .content > .feedback-logo, .content > .about-logo {
+    right: 10px;
+    left: initial;
+  }
+
+  html[dir="rtl"] .frontpage-alert {
+    margin-right: 20px;
+    margin-left: initial;
+  }
+
+  html[dir="rtl"] .content {
+    margin-right: 330px;
+    margin-left: initial;
+  }
+
+  html[dir="rtl"] .voclist #vocabulary-list {
+    margin-right: 20px;
+    margin-left: initial;
+  }
+  
+  html[dir="rtl"] .voclist-right #vocabulary-list {
+    margin-right: 20px;
+    margin-left: initial;
+  }
+
+  html[dir="rtl"] .voclist-left #vocabulary-list{
+    margin-right: 0;
+    margin-left: initial;
+  }
+
+  html[dir="rtl"] .voclist-left > .right-box {
+    left: 0;
+    right: initial;
+  }
+
+}
+
+@media (max-width: 1259px) {
+
+  html[dir="rtl"] .voclist > .right-box {
+    right: 0;
+    left: initial;
+  }
+
+}
+
+@media (min-width: 1260px) {
+  html[dir="rtl"] #service-name {
+    right: 0 !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/manual/NatLibFi/Skosmos/pullRequest?prid=4951454)